tools/java/org/unicode/cldr/tool/GenerateLanguageMatches.java

```diff
@@ -0,0 +1,105 @@
+package org.unicode.cldr.tool;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+
+import org.unicode.cldr.util.CLDRConfig;
+import org.unicode.cldr.util.CLDRFile;
+import org.unicode.cldr.util.StandardCodes;
+import org.unicode.cldr.util.StandardCodes.LstrField;
+import org.unicode.cldr.util.StandardCodes.LstrType;
+import org.unicode.cldr.util.SupplementalDataInfo;
+
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.Multimap;
+import com.google.common.collect.TreeMultimap;
+import com.ibm.icu.impl.Row.R4;
+import com.ibm.icu.text.Transform;
+
+public class GenerateLanguageMatches {
+    private static final CLDRFile ENGLISH = CLDRConfig.getInstance().getEnglish();
```

Comment:
These kinds of objects are inconvenient to debug when something goes wrong (e.g., file not found -- path setup problem).

Please make them local variables inside the main() function.

Reply:
This style is used all over CLDR, so I'd prefer consistency.

(If we decide to make a policy change and change, that's fine. That would involve changing a lot of tools from being static. )